.github/workflows/testing.yml

```diff
@@ -0,0 +1,22 @@
+name: Sync default to main
+on:
+  push:
+    branches:
+      - 'default'
+
+jobs:
+  sync:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v3
+        with:
+          fetch-depth: 0
+          token: ${{ secrets.CI_ACCESS_TOKEN }}
+      - name: Sync default branch to main
+        run: |
+          git config user.name "GitHub Actions"
+          git config user.email "<>"
```

Comment:
We can use the pattern here that we've used [elsewhere](https://github.com/pantheon-systems/wordpress-composer-managed/blob/default/devops/scripts/deploy-public-upstream.sh#L61-L62)

Reply:
Fixed